baselines/fedstar/README.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Federated Self-training for Semi-supervised Audio Recognition
+url: [dl.acm.org/doi/10.1145/3520128](https://dl.acm.org/doi/10.1145/3520128)
```

Comment:
```suggestion
url: https://dl.acm.org/doi/10.1145/3520128
```

Reply:
Thanks for the suggestion, updated the readme accordingly.